scripts/normalize_text.py

```
@@ -0,0 +1,94 @@
+    @staticmethod
+    def fix_file(path: Path) -> bool:
+        """Normalize a single file.
+
+        Returns:
+            True if file was modified, False otherwise.
+        """
+        if not path.is_file():
+            return False
+        try:
+            original = path.read_text(encoding="utf-8")
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)